.flake8

```diff
@@ -1,21 +1,27 @@
 [flake8]
-# E261: Adds no improvement to readability: Two spaces before inline comment
-# E123: flake8 is either buggy, or the rule is nonsensical. Doesn't work when nested: Closing brackets should match indentation of opening bracket.
-# E128: flake8 is either buggy, or the rule is nonsensical: Continuation line under-indented for visual indent
-# E221: Sometimes alignment of operators is better. Use common sense: "multiple spaces before operator"
-# E203: Sometimes alignment of commas is better. Use common sense: "whitespace before ','"
-# W503: Sometimes common sense prevail: "line break before binary operator"
-# W504: Sometimes common sense prevail: "line break after binary operator"
-# E226: "If operators with different priorities are used, __consider__ adding whitespace around the operators with the lowest priority(ies). Use your own judgment;..."
-# E722: Use common sense: "do not use bare 'except'"
-# E126: Do what feels right
-# E121: Do what feels right
-# E731: "E731 do not assign a lambda expression, use a def" -- don't tell me how to use lambdas
```

Comment:
I kinda liked the snarky comments ;)

Reply:
Haha it was funny at first but I kept losing track of what warning was which... also I’ve wanted to sort them for a while anyway.